graphprot/NeuralNet.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, database, Net,
         dataset = HDF5DataSet(root='./', database=database, index=self.index,
                               node_feature=self.node_feature, edge_feature=self.edge_feature,
                               target=self.target)
-        #PreCluster(dataset, method='mcl')
+        PreCluster(dataset, method='mcl')
```

Comment:
Indeed, my fault, it's a time consuming step that I always comment because it needs to be done only once on a dataset. But for users, better have it turned on automatically !